packages/salesforcedx-utils-vscode/test/jest/config/configUtil.test.ts

```diff
@@ -0,0 +1,16 @@
+import { ConfigUtil } from '../../../src';
+
+describe('ConfigUtil', () => {
+  describe('getProjectPackageNames', () => {
+    it('should return project package directories listed in project config file', () => {
+      // Need to stub:
+      // getRootWorkspacePath
+      // SfProject.getInstance()
+      // project.getUniquePackageNames()
+
+      const a = ConfigUtil.getProjectPackageNames();
+
+      expect(a).toEqual('');
```

Comment:
Started a test here, but need to figure out how to mock this stuff

Reply:
Is it weird to have this file be the same name as the existing ConfigUtil test file that lives in the test/unit dir?

Reply:
not weird.  The jest/unit location tells us the intention of each